installation-manager/src/main/resources/org/wildfly/core/instmgr/cli/command_resources.properties

```diff
@@ -13,16 +13,197 @@
+  List previous installation states.
+
+installer.history.option.revision.description=\
+  Hash of an installation state. When using the revision argument, a detailed description about the changes made in that revision will be displayed.
+
+# Revert
+installer.revert.description=\
+  Reverts to a previous installation state.\
+  \n\
+  \n\
+  Prepare-Revert: This phase prepares the installation you want to revert to. The candidate installation will be prepared on the target installation temporal directory \
+  (jboss.domain.temp.dir on domain mode or jboss.server.temp.dir on standalone server mode). Once this phase is completed, no further preparations can be made. \
+  If you decide to discard this candidate installation, you can use the 'clean' operation at any time. \
+  \n\
+  \n\
+  Apply-Revert: Once the candidate installation is prepared, you can apply it by using the 'shutdown --perform-installation' command. This command \
+  will restart the server and apply the candidate installation.\
+  \n\
+  \n\
+  If you are using a local interactive jboss-cli session, the session will be automatically closed. You will need to connect to the local server again. It will ensure \
+  that your new session uses the most recent changes available on the local installation.
+
+installer.revert.option.repositories.description=\
+  A collection of Maven repositories that contains the artifacts required to prepare an installation at the state specified by 'revision' argument. Multiple repositories are separated by comma. \
+  Valid values are either URLs or ID::URL pairs. This argument allows you to specify an alternative list of repositories for this revert command, \
+  replacing the repositories defined in the subscribed channels.
+
+installer.revert.option.local-cache.description=\
+  Path to the local Maven repository cache. It overrides the default Maven repository at ~/.m2/repository. The 'path' value represents the location within the remote machine's file system to which this session is connected.
+
+installer.revert.option.no-resolve-local-cache.description=\
+  Perform the operation without resolving or installing artifacts from/into local maven cache.
+
+installer.revert.option.offline.description=\
+  Perform installation from local or file-system Maven repositories only.
+
+installer.revert.option.maven-repo-file.description=\
+  The location of the local Maven Repository Zip file, containing the necessary artifacts to prepare the server installation we want to apply.
+
+installer.revert.option.revision.description=\
+  Hash of an installation state of the previous version you want to revert to.
+
+installer.revert.option.headers.description=\
+  Any Operation headers. Depending on the target environment, preparing a server could take some time because all required artifacts needs to be downloaded \
+  to provision the candidate installation. You can use Operation headers to increase the default blocking timeout which is 300 seconds.
```

Comment:
@bstansberry good suggestion, I did not think about any other specifically for this operation, so I think I can add a specific option to handle this.

Right now there is no restriction on the CLI side about the names; maybe the header to run the operation by a different role could be useful here? Otherwise, I can replace it with a specific option for the blocking-timeout